tests/pom.xml

```diff
@@ -91,6 +91,8 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
     <hono.deviceregistry.supportsGatewayMode>true</hono.deviceregistry.supportsGatewayMode>
     <!-- should be set to false if testing against a registry that doesn't support the "search Devices" operation -->
     <hono.deviceregistry.supportsSearchDevices>true</hono.deviceregistry.supportsSearchDevices>
+    <!-- should be set to true if testing against a registry that supports the "search Tenants" operation -->
+    <hono.deviceregistry.supportsSearchTenants>false</hono.deviceregistry.supportsSearchTenants>
```

Comment:
This has the opposite default value as the search devices property because the search tenants operation is implemented only in the _MongoDB registry_ and not in the default _file based device registry_. We don't have a separate _profile_ for the _file based device registry_ to mark this property as _false_. If we like to have the same default value then we need to create a new maven profile for the file-based device registry and set this property to `false` there. WDYT?

Reply:
Do we plan to implement this in the file based registry? If not, then we should probably should make the mongodb based registry the default one and replace the profile for the Mongo DB registry with a profile for the file based one ...

Reply:
Since we are planning to remove the _file based registry_ at some point of time, IMHO it doesn't make sense to implement this feature there. I will make the _MongoDB based registry_ as the default one.

Reply:
make sure to also adapt the GitHub actions workflow accordingly ...